x/x.go

```diff
@@ -1135,12 +1135,11 @@ func IsGuardian(groups []string) bool {
 	return false
 }
 
-// RunVlogGC runs value log gc on store. It runs GC unconditionally after every 10 minutes.
-// Additionally it also runs GC if vLogSize has grown more than 1 GB in last minute.
+// RunVlogGC runs value log gc on store. It runs GC unconditionally after every 1 minute.
 func RunVlogGC(store *badger.DB, closer *z.Closer) {
 	defer closer.Done()
 
-	// Runs every 1m, checks size of vlog and runs GC conditionally.
```

Comment:
Conditionally on what?  Can be good to make that clear.  I believe is the discard ratio (looks like 0.7 in `err = store.RunValueLogGC(0.7)` below).  We can make a quick note that this is a number we chose based on performance considerations.

Reply:
It no longer runs conditionally now. And the discard ratio is well documented in the `badger` code. 

For the discard ratio number, it's quite well documented in the `badger` code, and we don't really know how the old team picked `0.7` per se. We only know that on paper, setting it to 0.7 means a file will be rewritten if at least 70% of the space can be discarded. This means fewer space reclaims will happen which means less background activity. So I think it wouldn't add much value as of now if we add it to the comments.

Reply:
I am sorry, I completely missed that your comment says "unconditionally" 😄 so this looks good.

Reply:
haha no worries Josh, but I do think we need to benchmark that `0.7` discard ratio "magic number" some time in the future, see if it really is the optimal value for our ideal case workload.